client/trino-cli/pom.xml

```diff
@@ -72,6 +72,12 @@
             <artifactId>opencsv</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>net.steppschuh.markdowngenerator</groupId>
+            <artifactId>markdowngenerator</artifactId>
```

Comment:
do we need this dependency? can we provide our own? how hard would it be?

Reply:
Nevermind, i've just checked that this is standalone jar without further dependencies that is 36 KB so fine to add.